### PR TITLE
152 move eit data loading

### LINKED ISSUE #152: Decrease the number of levels of hierarchy for `EITData` subclasses.
Currently, `EITData_` is the direct parent for individual vendor EITData classes (e.g. `DraegerEITData`) and itself inherits from `EITData`. It would be nice if we can find a way to circumvent this intermediate class. However, that would lead to the following issue:

> `EITData.from_path()` requires `vendor` as argument, so it can choose the right class to load the file. 
> `DraegerEITData.from_path()` does not allow `vendor` as argument, because the vendor is implicit in the choice of class. Of course the same is true for SentecEITData and TimpelEITData.
> It is possible to override `from_path()` in each of the three subclasses, but that would not be DRY. 
> 
> `EITData_` removes `vendor` from the initializer with `field(init=False)`. 
> Each subclass sets `vendor` to the proper value.
> `EITData_` also changes the arguments required for `from_path()`.
> 
> I wasn't able to solve this without an intermediate class. Any other solution I came up with requires re-implementing `from_path()` in each of the three subclasses.

_Originally posted by @psomhorst in https://github.com/EIT-ALIVE/eitprocessing/pull/142#discussion_r1514386680_

### PR DESCRIPTION
**Pull request by psomhorst**
Closes #152 

@DaniBodor This is a quick idea for reducing the hierarchy levels of EITData. The (simple?) solution is to remove the vendor specific classes entirely and replace them with modules containing vendor-specific loading functions. A general loading function is still available, which is probably the one people will use most often. This splits the responsibility for handling data/general usage (EITData class) from loading the data (loading module). 

The interface becomes:

```python
from eitprocessing.eit_data.loading import load_from
eit = load_from(<path>, vendor="sentec")  # returns a DataCollection
plt.plot(eit["raw"].global_impedance)
```

or 

```python
from eitprocessing.eit_data.loading import draeger
from pathlib import Path
eit = draeger.load_from_single_path(Path(<path>))  # returns a DataCollection
```

I didn't have to do much for this change. The functions loading the data were already quite independent from the class itself. 

What do you think?

### CODE REVIEW

File: eitprocessing/eit_data/loading/draeger.py [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    from pathlib import Path
+
+    from numpy.typing import NDArray
+
+_FRAME_SIZE_BYTES = 4358
+FRAMERATE = 20
+
+
+def load_from_single_path(
+    path: Path,
```
Comment (DaniBodor) [inline eitprocessing/eit_data/loading/draeger.py:28]: why do we not allow multiple paths here, as we do for the generic loading function?

File: eitprocessing/eit_data/loading/draeger.py [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+from eitprocessing.eit_data.vendor import Vendor
+from eitprocessing.sparse_data import SparseData
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from numpy.typing import NDArray
+
+_FRAME_SIZE_BYTES = 4358
+FRAMERATE = 20
```
Comment (DaniBodor) [inline eitprocessing/eit_data/loading/draeger.py:24]: To avoid confusion between modules, how about 

```suggestion
DRAEGER_FRAMERATE = 20
```

**Review by DaniBodor — COMMENTED** [review]
I assume the functions themselves haven't changed from the previous setup. git isn't parsing them nicely for me to see what changed line-by-line and don't want to go through it by eye to compare precisely if not needed. Can you confirm that this is the case.

I agree that it's a good idea to deal with `return_non_eit_data`, but doesn't necessarily need to be in the same PR.

Apart from that, just few minor comments.

File: eitprocessing/eit_data/loading/sentec.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+from eitprocessing.eit_data import EITData
+from eitprocessing.eit_data.vendor import Vendor
+from eitprocessing.sparse_data import SparseData
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from numpy.typing import NDArray
+
+VENDOR = Vendor.SENTEC
```
Comment (DaniBodor) [inline eitprocessing/eit_data/loading/sentec.py:23]: I don't think this is used anywhere

```suggestion
```

**Comment by psomhorst** [comment]
@DaniBodor I think I changed all things we talked about.

**Review by DaniBodor — APPROVED** [review]

**Pull request merged**